content/en/docs/setup/upgrade/istioctl-upgrade/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Upgrade and downgrade Istio using istioctl [Experimental]
+description: Upgrade or downgrade Istio using the istioctl upgrade command.
+weight: 25
+aliases:
+    - /docs/setup/kubernetes/upgrade/steps/
+    - /docs/setup/upgrade/steps
+keywords: [kubernetes,upgrading]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+The `istioctl experimental upgrade` command performs an upgrade of Istio. Before performing
+the upgrade, it checks that the Istio installation meets the upgrade eligibility
+criteria. Also, it alerts the user if it detects any changes in the profile
+default values between Istio versions.
+
+The upgrade command can also perform a downgrade of Istio.
+
+See the [`istioctl` upgrade reference](/docs/reference/commands/istioctl/#istioctl-experimental-upgrade)
+for all the options provided by the `istioctl experimental upgrade` command.
+
+## Upgrade prerequisites
+
+Ensure you meet these requirements before starting the upgrade process:
+
+* Istio version 1.3.3 or higher is installed.
+
+* Your Istio installation was installed using the
+   [operator CLI command](/docs/setup/install/operator/).
+
+## Upgrade steps
+
+{{< warning >}}
+Traffic disruption may occur during the upgrade process. To minimize the disruption, ensure
+that at least two replicas of each component (except Citadel) are running. Also, ensure that
+[`PodDistruptionBudgets`](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)
+are configured with a minimum availability of 1.
+{{< /warning >}}
+
+1. [Download the new Istio release](/docs/setup/#downloading-the-release)
+   and change directory to the new release directory.
+
+1. Verify that `istoctl` supports upgrading to your intended target Istio version by
+   viewing the supported versions list:
+
+    {{< text bash >}}
+    $ istioctl manifest versions
+    {{< /text >}}
+
+1. Ensure that your Kubernetes configuration points to the cluster to upgrade:
+
+    {{< text bash >}}
+    $ kubectl config view
+    {{< /text >}}
+
+1. Begin the upgrade by running this command:
+
+    {{< text bash >}}
+    $ istioctl experimental upgrade -f `<your-custom-configuration-file>`
```

Comment:
Perhaps add below: "your-custom-configuration-file" is the file you used to install the currently running version of Istio.